dm/pkg/conn/baseconn.go

```diff
@@ -23,14 +23,14 @@ import (
 	gmysql "github.com/go-mysql-org/go-mysql/mysql"
 	"github.com/go-sql-driver/mysql"
 	"github.com/pingcap/failpoint"
+	"github.com/pingcap/tiflow/dm/pkg/utils"
```

Comment:
import groups messed up, other files too

can set per-project import group setting in goland